oxd-server/src/test/java/org/gluu/oxd/server/GetAuthorizationUrlTest.java

```diff
@@ -29,4 +31,18 @@ public void test(String host, String redirectUrl, String opHost) {
         assertNotNull(resp);
         notEmpty(resp.getAuthorizationUrl());
     }
+
+    @Parameters({"host", "redirectUrl", "opHost", "redirectUrls", "postLogoutRedirectUrl", "logoutUrl", "paramRedirectUrl"})
+    @Test
+    public void testWithParameterAuthorizationUrl(String host, String redirectUrl, String opHost, String redirectUrls, String postLogoutRedirectUrl, String logoutUrl, String paramRedirectUrl) throws IOException {
+        ClientInterface client = Tester.newClient(host);
+
+        final RegisterSiteResponse site = RegisterSiteTest.registerWithMultipleRedirectUrls(client, opHost, redirectUrl, postLogoutRedirectUrl, logoutUrl, redirectUrls);
+        final GetAuthorizationUrlParams commandParams = new GetAuthorizationUrlParams();
+        commandParams.setOxdId(site.getOxdId());
+        commandParams.setAuthorizationRedirectUri(paramRedirectUrl);
+        final GetAuthorizationUrlResponse resp = client.getAuthorizationUrl(Tester.getAuthorization(), commandParams);
+        assertNotNull(resp);
+        notEmpty(resp.getAuthorizationUrl());
```

Comment:
check here whether `paramRedirectUrl` is contained by `resp.getAuthorizationUrl()`

Reply:
Changes done.